<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>work_sen</class>
 <widget class="QWidget" name="work_sen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>352</y>
     <width>173</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="on_true">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>298</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="on_false">
   <property name="geometry">
    <rect>
     <x>483</x>
     <y>298</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="water_set">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="water_have">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="have_true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>298</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="have_false">
   <property name="geometry">
    <rect>
     <x>164</x>
     <y>298</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="water_proc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>55</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="water_norm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="water_on">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_proc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>102</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_on">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>70</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="gas_set">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_have">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>70</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="gas_norm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>100</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp_proc">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>152</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp_on">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="temp_set">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>29</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp_have">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>130</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="temp_norm">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="simulat">
   <property name="geometry">
    <rect>
     <x>647</x>
     <y>298</y>
     <width>157</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="roomlist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>181</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-top-color: rgb(92, 53, 102);
selection-background-color: rgb(173, 127, 168);
color: rgb(238, 238, 236);
background-color: rgba(173, 127, 168,0.1);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cond_have_true">
   <property name="geometry">
    <rect>
     <x>225</x>
     <y>236</y>
     <width>173</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="cond_on">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>180</y>
     <width>591</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cond_have_false">
   <property name="geometry">
    <rect>
     <x>415</x>
     <y>236</y>
     <width>173</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cond_on_but">
   <property name="geometry">
    <rect>
     <x>604</x>
     <y>236</y>
     <width>173</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:3px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>203</x>
     <y>274</y>
     <width>601</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="energy">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>202</y>
     <width>54</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(238, 238, 236);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>400</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Acrom Medium</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="pich.qrc">:/picha/Pichures/work_sen.png</pixmap>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>back</zorder>
  <zorder>on_true</zorder>
  <zorder>on_false</zorder>
  <zorder>water_set</zorder>
  <zorder>water_have</zorder>
  <zorder>have_true</zorder>
  <zorder>have_false</zorder>
  <zorder>water_proc</zorder>
  <zorder>water_norm</zorder>
  <zorder>water_on</zorder>
  <zorder>gas_proc</zorder>
  <zorder>gas_on</zorder>
  <zorder>gas_set</zorder>
  <zorder>gas_have</zorder>
  <zorder>gas_norm</zorder>
  <zorder>temp_proc</zorder>
  <zorder>temp_on</zorder>
  <zorder>temp_set</zorder>
  <zorder>temp_have</zorder>
  <zorder>temp_norm</zorder>
  <zorder>simulat</zorder>
  <zorder>roomlist</zorder>
  <zorder>cond_have_true</zorder>
  <zorder>cond_on</zorder>
  <zorder>cond_have_false</zorder>
  <zorder>cond_on_but</zorder>
  <zorder>warning</zorder>
  <zorder>energy</zorder>
 </widget>
 <resources>
  <include location="pich.qrc"/>
 </resources>
 <connections/>
</ui>
